base/applications/mspaint/main.cpp

```diff
@@ -300,7 +300,16 @@ _tWinMain (HINSTANCE hThisInstance, HINSTANCE hPrevInstance, LPTSTR lpszArgument
     sfn.nMaxFileTitle  = SIZEOF(sfnFiletitle);
     sfn.Flags          = OFN_EXPLORER | OFN_OVERWRITEPROMPT | OFN_HIDEREADONLY | OFN_EXPLORER | OFN_ENABLEHOOK;
     sfn.lpfnHook       = OFNHookProc;
-    sfn.lpstrDefExt    = L"bmp";
+    sfn.lpstrDefExt    = L"png";
+    // Choose PNG
+    for (INT i = 0; i < aguidFileTypesE.GetSize(); ++i)
```

Comment:
It would be nice if the default filename translations didn't contain the file extension, but instead just the filename without it. The extension being appended by either the save-dialog and/or here in code.

Reply:
Done. https://github.com/reactos/reactos/commit/50cb4b3cb76da023bb83676fd4dc4e6b2f116aaa